binding/java/build.gradle

```diff
@@ -105,6 +105,10 @@ task sourceJar(type: Jar, dependsOn: classes) {
     }
 }
 
+javadoc {
+    failOnError = false
```

Comment:
Otherwise "./gradle" command always fails for me. It reports that many
javadocs are invalid.
On Aug 11, 2015 3:22 PM, "Ivan De Marino" <notifications@github.com> wrote:

> In binding/java/build.gradle
> <https://github.com/detro/ghostdriver/pull/462#discussion_r36738269>:
>
> > @@ -105,6 +105,10 @@ task sourceJar(type: Jar, dependsOn: classes) {
> >      }
> >  }
> >
> > +javadoc {
> > +    failOnError = false
>
> why this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/detro/ghostdriver/pull/462/files#r36738269>.
>